client/src/dbus/api/secret.rs

```diff
@@ -5,15 +5,20 @@ use zbus::zvariant::{OwnedObjectPath, Type};
 use zeroize::{Zeroize, ZeroizeOnDrop};
 
 use super::Session;
-use crate::{crypto, dbus::Error, Key};
+use crate::{crypto, dbus::Error, Key, Secret};
 
-#[derive(Debug, Serialize, Deserialize, Type)]
+#[derive(Debug, Serialize, Deserialize, Type, Zeroize, ZeroizeOnDrop)]
 #[zvariant(signature = "(oayays)")]
-pub struct SecretInner(pub OwnedObjectPath, pub Vec<u8>, pub Vec<u8>, pub String);
+pub struct SecretInner(
+    #[zeroize(skip)] pub OwnedObjectPath,
+    pub Vec<u8>,
+    pub Vec<u8>,
```

Comment:
Why do we need to Zeroize here now? These (the vecs) are just encrypted (and an IV) bytes.

I think with this change I'll have to do some kind of conversion when accessing the SecretInner attribute in server side. Specifically when calling a function such as `oo7::crypto::encrypt()`.

Reply:
What kind of conversion? all the crypto methods takes AsRef<[u8]> which is implemented by the zeroized types, so nothing to do here.

Reply:
And no, Zeroize should always been there, but probably worth moving into a separate commit.

Reply:
see my comment on the other review